### PR TITLE
Update homu links.

### PR DESCRIPTION
**Pull request by luketheobscure**
http://homu.io/ isn't up anymore.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ Before the review period starts, the previous version should be re-released with
 
 # Pull Requests
 
-You should use [homu](http://homu.io) when accepting pull requests.
+You should use [homu](https://github.com/barosl/homu) when accepting pull requests.
 You can read about the available commands in the front page.
 
-The Guides repository has homu [configured to auto-squash commits](http://homu.io/r/emberjs/guides).
+The Guides repository has homu configured to auto-squash commits.
```
Comment (luketheobscure) [inline MAINTAINERS.md]: Is there a different URL this should point to?

Comment (locks) [inline MAINTAINERS.md]: We don't actually use any bot at the moment, though we could try setting up bors-ng.

Comment (luketheobscure) [inline MAINTAINERS.md]: Then should I just remove references to homu?

Comment (locks) [inline MAINTAINERS.md]: I think we should, and we can update if we start using bors.

Comment (luketheobscure) [inline MAINTAINERS.md]: All references to homu have been removed.

**Pull request merged**